src/current/v23.2/work-with-virtual-clusters.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Work with virtual clusters
+summary: Learn how to connect to and work with virtual clusters.
+toc: true
+docs_area: deploy
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Refer to the [Cluster Virtualization Overview]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}#known-limitations) for further detail.
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} This page shows how to work with a cluster with [cluster virtualization]({% link {{ page.version.version }}/cluster-virtualization-overview.md %}).
+
+## Connect to a virtual cluster
+
+This section shows how to use SQL clients or the DB Console to connect to a virtual cluster.
+
+{% capture pcr_application_cluster_note %}
+{{site.data.alerts.callout_success}}
+When [Physical Cluster Replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) is enabled, the default virtual cluster is named `application`.
+{{site.data.alerts.end}}
+{% endcapture %}
+
+{{ pcr_application_cluster_note }}
+
+### DB Console
+
+This section shows how to connect using DB Console when cluster virtualization is enabled.
+
+{{site.data.alerts.callout_success}}
```

Comment:
General comment that this page is quite heavy on the tips/notes, I wonder if some of them could be consolidated into the body of the text.